src/components/Home/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { redditData } from '../../API/contants'
+import API from '../../API'
+
+const Home = () => {
+
+    const getRedditData = () => {
+        API.get(redditData).then(result => {
```

Comment:
We can directly export functions from a `.ts` file. Then we can avoid chaining the function call here. What do you think ?

Reply:
I didn't get ur point. which function u r talking about?

Reply:
I'm talking about `API/index.ts` file. We can directly export `get` function from index.ts file and can use `named imports` here in this file.